docs/modules/ROOT/pages/delete-account.adoc

```diff
@@ -1,36 +1,52 @@
-= Deleting an Account
-:description: pass:q[If you no longer need to access your clusters, you can delete your account. To avoid additional charges while your clusters are not in use, consider xref:stop-and-resume.adoc[pausing] them instead.]
-:cloud-tags: Manage Accounts
-:cloud-title: Deleting an Account
+= Delete Organization
+:description: pass:q[If you no longer need to access your clusters, you can delete your organizaton. To avoid additional charges while your clusters are not in use, consider xref:stop-and-resume.adoc[pausing] them instead.]
+:cloud-tags: Organizations and Accounts
+:cloud-title: Delete Organization
 :cloud-order: 60
 :toclevels: 3
 
 {description}
 
+You cannot delete an organization while there are outstanding payments on the account.
 
-[[cluster-and-data-deletion]]
-== Cluster and Data Deletion
+If the organization is deleted, all associated users, clusters, and resources are also deleted. If you need to keep any data, ensure that you have migrated it before deleting the organization.
 
-Deleting an account _removes_ all the following:
+NOTE: You can remove organization users from your organization at any time through the User Management screen. For further information on removing organization users, see the xref:remove-user.adoc[Remove User] topic.
 
-- Clusters linked to the account, including any associated data, such as data structures, custom classes, backups, and configuration.
-- Account and payment details.  
-- Billing information.
+Deleting an organization removes the following:
 
-We recommend that you copy any data or details that you might need in the future. You cannot recover any data from a deleted account.
+- Clusters associated with the organization, including any data such as data structures, custom classes, backups, and configuration
+- Account and payment details
+- Billing information
+- The admin user account
+- All organization user sign ins
 
-NOTE: You may still receive marketing communications from time to time. See our link:https://hazelcast.com/privacy/[privacy policy] for more details.  
+NOTE: You might continue to receive occasional marketing communications from Hazelcast. See our link:https://hazelcast.com/privacy/[privacy policy^] for more details.  
 
-[[outstanding-payments]]
-== Outstanding Payments
+Hazelcast recommends that you copy any data or details that you might need in the future. You cannot recover any data from a deleted account.
 
-All outstanding payments are taken during the deletion process. For this reason, we suggest that you pause or delete all clusters before you proceed, so that you know the amount you need to pay.
+To delete an organization, complete the following steps:
 
-== Deleting your Account
+. Open the link:{page-cloud-console}[Hazelcast {hazelcast-cloud} console,window=_blank].
+. Select *Account* from the side navigation bar
```

Comment:
I believe the option to delete the org is moving to the more logical position in the Organization options.